kyo-core/shared/src/test/scala/kyo/AsyncTest.scala

```diff
@@ -519,7 +519,7 @@ class AsyncTest extends Test:
             yield assert(result == 42)
         }
 
-        "interrupts computation" in run {
+        "interrupts computation" in runJVM {
```

Comment:
Unrelated to the `Clock` changes. This test sometimes fails in JS, I think the issue is the ordering of the execution of the callbacks that is not deterministic.